### PR TITLE
[ADD] ansible-swarm-aws: Create Ansible role

### PR DESCRIPTION
**Pull request by tedsalmon**
* Role launches EC2 nodes for Swarm
* Role provisions Swarm Manager + Swarm Worker Nodes

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+galaxy_info:
+  author: your name
```
Comment (lasley) [inline meta/main.yml]: Update this file

**Comment by lasley** [comment]
@yajo - not sure of your tech stack, but you may be interested in this.

**Review by lasley — APPROVED** [review]
Great job dude. One note on the Galaxy meta, but everything else looks good.

**Comment by lasley** [comment]
@tedsalmon - can we add EC2 tags through this? It would be nice to have a three vars, one to apply to all and one for each swarm master/slave

**Comment by lasley** [comment]
Also, how to target specific VPCs?

**Comment by tedsalmon** [comment]
@lasley,

Both requests have been attended to in commit cd1bc08 :)

Thanks!

**Comment by yajo** [comment]
Maybe you'll think I'm a fool, but I never used ansible before. I'll keep this as a reference just in case. 😊

**Comment by lasley** [comment]
It's the best infrastructure management tool out there IMO. If you've ever used Puppet, it's similar to that except way more badass.

You may be able to get around the need somewhat with a tool like Rancher, but I've found that it's much better to have a separate system for that. Basically your Ansible will control the metal (or VMs) & Docker controls everything else.

Take a look at our [ansible_base](https://github.com/LasLabs/ansible-base), which is a playbook that contains a few roles & really demonstrates the power of Ansible. My particular favorite is the [RHEL7-STIG](https://github.com/MindPointGroup/RHEL7-STIG/tree/b344671f7c8fe548f5f04d38c93bcaf576ef2f10) role that is included as a submodule.

**Pull request merged**